src/app/plan/duck/selectors.ts

```diff
@@ -116,11 +116,28 @@ const getPlansWithPlanStatus = createSelector(
       const isPlanLocked = !!lockedPlans.some(
         (lockedPlan) => lockedPlan === plan.MigPlan.metadata.name
       );
-      const latestMigration = plan.Migrations.length ? plan.Migrations[0] : null;
-      const latestAnalytic = plan.Analytics?.length ? plan.Analytics[0] : null;
-      const latestType = latestMigration?.spec?.stage ? 'Stage' : 'Migration';
 
-      const hasSucceededStage = !!plan.Migrations.filter((m) => {
+      // Rollback acts as a "reset", so we should only consider migrations after rollback
+      let planMigrations;
+      if (plan.Migrations.length) {
+        const latestRollbackIndex = plan.Migrations.findIndex((m) => {
+          return m.spec.rollback && m.status.conditions.some((c) => c.type === 'Succeeded');
+        });
+        planMigrations = plan.Migrations.slice(0, latestRollbackIndex + 1);
```

Comment:
The major change this PR makes is that after rollback occurs, the list of `MigMigrations` considered when evaluating `hasMigrationSucceeded` etc. is limited to those that have taken place _after_ rollback.